pyproject.toml

```diff
@@ -25,7 +25,7 @@ packages = [{ include = "data_diff" }]
 [tool.poetry.dependencies]
 python = "^3.7"
 runtype = "^0.2.6"
-dsnparse = "*"
+dsnparse = "<0.2.0"
```

Comment:
The reason that `dsnparse 0.2.0` uses an operator `:=` so tests for Python 3.7 [failed](https://github.com/datafold/data-diff/actions/runs/4667114880/jobs/8262492651#step:8:277) because of that. Maybe that is why they have increased the major minor version from 1 to 2.

Reply:
Thank you very much! It definitely looks better than just using one hard filter.

Reply:
Done.